lib-core/src/main/java/com/kinandcarta/create/proxytoggle/lib/core/android/DeviceSettingsManager.kt

```diff
@@ -1,17 +1,20 @@
-package com.kinandcarta.create.proxytoggle.manager.android
+package com.kinandcarta.create.proxytoggle.lib.core.android
```

Comment:
this class knows about Android? if lib core is intended to be used by the entire app, this should be in here
unless you are not trying to do clean architecture, then it doesn't matter

Reply:
Yeah this class is used in both the widget and the app. It's the one that does all the job tbh